.github/workflows/linting.yml

```diff
@@ -0,0 +1,30 @@
+name: Linting
+
+on:
+  push:
+    branches: [ development ]
+  pull-request:
+    branches: [development]    
+
+  
+jobs:
+  linting:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        
+    
+      - name: Extra dependencies
+        run: |
+          sudo apt install clang-format
+          
+
+      - name: Run linting
+        run: | 
+          find ../ -regex '.*\.\(cc\|h\)' -not -iname '*.in.h' | xargs -I{} -P 10 clang-format -i -style=file {}; git diff
```

Comment:
```suggestion
          find ../ -regex '.*\.\(cc\|h\)' -not -iname '*.in.h' | xargs -I{} -P 10 clang-format -i -style=Google; git diff
```

Reply:
@dpshelio : Keep using `file`, we have some custom changes to the style and we want to keep control of the version to ourselves.